playwright/pages/blocks-dashboard.page.ts

```diff
@@ -1,15 +1,252 @@
+    ).toBeGreaterThanOrEqual(blockFrostBlock?.confirmations);
+
+    expect(
+      +((await blocksDetailTotalTrx.textContent()) || 0),
+      "Total transaction in block to greate or equal total transaction in Blockfrost"
+    ).toBeGreaterThanOrEqual(blockFrostBlock?.confirmations);
+
+    expect(
+      +((await blocksDetailTrxFee.textContent())?.replaceAll(",", "") || 0) * 10 ** 6,
+      "Total fees in block detail to equal total fee in block on Blockfrost "
+    ).toEqual(+(blockFrostBlock?.fees || 0));
+
+    expect(
+      +((await blocksDetailOutput.textContent())?.replaceAll(",", "") || 0) * 10 ** 6,
+      "Total output in block detail to equal total output in block on Blockfrost "
+    ).toEqual(+(blockFrostBlock?.output || 0));
+  };
+
+  const checkTransactionsTable = async () => {
+    await expect(blocksDetailTrxTableTxhash, "Check title on transaction table").toHaveText("Tx hash");
+    await expect(blocksDetailTrxTableCreatedAt, "Check title on transaction table").toHaveText("Created At");
+    await expect(blocksDetailTrxTableAddress, "Check title on transaction table").toHaveText("Addresses");
+    await expect(blocksDetailTrxTableFees, "Check title on transaction table").toHaveText("Fees");
+    await expect(blocksDetailTrxTableoutput, "Check title on transaction table").toHaveText("Output in ADA");
+  };
+
+  const checkBlockWidget = async ({ blockFrostBlock }: { blockFrostBlock: BlockInformationDto }) => {
+    expect(+((await blocksWidgetEpochNo.textContent()) || 0), "Check epoch on block widget").toEqual(
+      blockFrostBlock.epoch
+    );
+    expect(
+      +((await blocksWidgetSlotNo.textContent())?.split("/")[0] || 0),
+      "Slot no in block widget to equal total fee in block on Blockfrost "
+    ).toEqual(+(blockFrostBlock?.epoch_slot || 0));
+
+    expect(
+      +((await blocksWidgetAbsSlotNo.textContent()) || 0),
+      "Absolute slot in block widget to equal total fee in block on Blockfrost "
+    ).toEqual(+(blockFrostBlock?.slot || 0));
+
+    expect(
+      moment((await blocksWidgetCreatedAt.textContent())?.replace(",", "")).unix(),
+      "Created time to equal create time epoch Blockfrost"
+    ).toEqual(blockFrostBlock?.time);
+
+    expect(
+      +((await blocksWidgetConfirmation.textContent()) || 0),
+      "Confirmation in block widget to equal total fee in block on Blockfrost "
+    ).toBeGreaterThanOrEqual(+(blockFrostBlock?.confirmations || 0));
+
+    expect(
+      +((await blocksWidgetFee.textContent())?.replaceAll(",", "") || 0) * 10 ** 6,
+      "Total fees in block widget to equal total fee in block on Blockfrost "
+    ).toEqual(+(blockFrostBlock?.fees || 0));
+
+    expect(
+      +((await blocksWidgetOutput.textContent())?.replaceAll(",", "") || 0) * 10 ** 6,
+      "Total output in block widget to equal total fee in block on Blockfrost "
+    ).toEqual(+(blockFrostBlock?.output || 0));
+  };
+
+  const checkTransactionsDetail = async () => {
```

Comment:
When we check the redirect behavior as mention for the epochs tests we should also make a simple validation that the redirect goes to the respective element, in this case the user selects the transaction hash so when he be redirected we should also compare the selected tx hash is the same that is displayed in the transaction detail page and like mention for epochs test this check is executed over the transaction detail page so this function and respective locator should be in a separated page called transaction-detail.page.ts for example.